src/scss/colorways.scss

```diff
@@ -467,6 +467,50 @@
 .gmk-dolch-kb {
 }
 
+.gmk-dualshot-key {
+  background: darken($color-gmk-abs-2B, 5%);
+  color: darken($color-gmk-abs-N9, 12%);
+}
+.gmk-dualshot-mod {
+  background: darken($color-gmk-abs-2B, 5%);
+  color: darken($color-gmk-abs-N9, 12%);
+}
+.gmk-dualshot-mod2 {
+  background: darken($color-gmk-abs-2B, 5%);
+  color: lighten($color-gmk-abs-RO2, 35%);
```

Comment:
Try this and let me know how you like it. It's still fairly pink but not as pink as current.

```suggestion
  color: lighten($color-gmk-abs-RO2, 22%);
```

Reply:
Can i see the preview @noroadsleft?

Reply:
![image](https://user-images.githubusercontent.com/18669334/95160530-b0843100-0755-11eb-9a5d-9f96ff2ba55e.png)

Reply:
What if `color: darken($color-gmk-abs-RO2, 5%);` @noroadsleft? I think it's important to make it still red.

Reply:
Unfortunately I don't find it legible at `color: darken($color-gmk-abs-RO2, 5%);`:

![image](https://user-images.githubusercontent.com/18669334/95362502-060a2c00-0883-11eb-88ac-fadd2f0e0f4a.png)

Maybe try `color: saturate(lighten($color-gmk-abs-RO2, 13%), 12%);` (Line 480):

![image](https://user-images.githubusercontent.com/18669334/95365755-69965880-0887-11eb-8f02-e5ea5c277c49.png)

Reply:
Ok, `color: saturate(lighten($color-gmk-abs-RO2, 13%), 12%);` awesome.